test/iron-request.html

```diff
@@ -115,11 +115,28 @@
 
         });
 
+        test('default responseType of text is not applied, when async is false', function (done) {
+
+          var options = Object.create(successfulRequestOptions);
+          options.async = false;
+
+          request.send(options);
+          server.respond();
+
+          request.completes.then(function() {
```

Comment:
Prefer returning a promise rather than calling done explicitly in tests. This means that you can also eliminate your call to catch.